src/status_im2/contexts/quo_preview/list_items/dapp.cljs

```diff
@@ -0,0 +1,61 @@
+(ns status-im2.contexts.quo-preview.list-items.dapp
+  (:require [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im2.common.resources :as resources]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label   "State"
+    :key     :state
+    :type    :select
+    :options [{:key   :default
+               :value "Default"}
+              {:key   :pressed
+               :value "Pressed"}
+              {:key   :active
+               :value "Active"}]}
+   {:label   "Action:"
+    :key     :action
+    :type    :select
+    :options [{:key   :none
+               :value "None"}
+              {:key   :icon
+               :value "Icon"}]}
+   {:label "Blur?"
+    :key   :blur?
+    :type  :boolean}])
+
+(defn cool-preview
+  []
+  (let [state (reagent/atom {:dapp                {:avatar (resources/get-mock-image :coin-gecko)
+                                                   :name   "Coingecko"
+                                                   :value  "coingecko.com"}
+                             :state               :default
+                             :action              :icon
+                             :blur?               false
+                             :customization-color :blue
+                             :on-press            (fn [] (js/alert "Button pressed"))})]
+    (fn []
+      [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
+       [rn/view {:padding-bottom 150}
+        [rn/view {:flex 1}
+         [preview/customizer state descriptor]]
+        [rn/view
+         {:padding-vertical 60
+          :flex-direction   :row
+          :justify-content  :center}
+         [quo/dapp @state]]]])))
+
+(defn preview
+  []
+  [rn/view
+   {:background-color (colors/theme-colors colors/white
+                                           colors/neutral-95)
+    :flex             1}
+   [rn/flat-list
+    {:flex                         1
+     :keyboard-should-persist-taps :always
+     :header                       [cool-preview]
+     :key-fn                       str}]])
```

Comment:
Thanks to Icaro! The preview boilerplate is no longer necessary. Please take a look at this PR for reference:  https://github.com/status-im/status-mobile/pull/16996

Reply:
awesome! done.